src/os_auth/config.c

```diff
@@ -56,16 +56,9 @@ cJSON *getAuthdConfig(void) {
     cJSON *disconnected_time = cJSON_CreateObject();
 
     cJSON_AddNumberToObject(auth,"port",config.port);
-    if (config.force_options.connection_time ==  -1)
```

Comment:
This file contains a reference to `config.force_options.connection_time = -1;` that we should remove